PokeStar/PokeStar/Modules/DexCommands.cs

```diff
@@ -86,7 +86,7 @@ public async Task CP([Remainder] string text)
 
             await Context.Channel.SendFileAsync(fileName, embed: embed.Build()).ConfigureAwait(false);
 
-            Connections.CopyFile(fileName);
```

Comment:
Why was this removed, is needed for pictures

Reply:
Should be changed to DeleteFile

Reply:
You wrote it that way and I didn't really understand it but removing it didn't break anything. I will change it to DeleteFile